LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -93,10 +99,22 @@ struct FHIRResourcesView: View {
             EmptyView()
         }
     }
-    
+
+    private var filteredResourceKeys: [String] {
```

Comment:
There is some duplication with the search clause in the `ForEach` here as we have the same search methodology here. Might be interesting if we can either put it in an extension or just a private function in the view so we don't duplicate the logic for the resource type and resources filtering.